src/applications/personalization/profile360/sass/address-validation.scss

```diff
@@ -0,0 +1,3 @@
+.address-validation-modal .va-modal-inner {
+  max-width: 50rem;
```

Comment:
Overriding the defaults this way seems unnecessary and error prone.  For example, if at some point in the future, the `<Modal>` doesn't rely on the `va-modal-inner` class, this will no longer work.

I understand that the default 40rem is smaller, but what kinds of problems does this cause?

Reply:
@bkjohnson I can remove it.  It was only added to get the output to look more like the mocks.